### DISCUSSION TITLE
Plug-ins

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#95] [Q&A]
@URUWorks, what do you think about allowing users to load and even create plug-ins? Is it technically possible, or are there obstacles? Asking as a reference not only for us, but for the benefit of others who might wish for it.

**Comment by URUWorks** [comment]
It is possible, perhaps the best way (and compatible with all systems) is using scripts as in SW.

**Comment by chenlung** [comment]
What about macros? I haven't used them.
